spec/support/draft_helpers.rb

```diff
@@ -101,6 +101,16 @@ def publish_variable_draft(provider_id: 'MMT_2', native_id: nil, name: nil, long
       end
     end
 
+    def add_variable_collection_associations(variable_id, collection_concept_ids)
```

Comment:
I added a similar helper for this in MMT-1124 (and it just got merged). I'm not sure if this PR is the right place to consolidate the two or if we should do that in a future PR.

Reply:
Sure, I'll take a look